<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgPrefLibraryDlg</class>
 <widget class="QWidget" name="DlgPrefLibraryDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Library Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxLibrary">
     <property name="title">
      <string>Library</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="TextLabel3_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>10</height>
         </size>
        </property>
        <property name="font">
         <font/>
        </property>
        <property name="text">
         <string>Music Directories:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>dirList</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="PushButtonRemoveDir">
        <property name="font">
         <font/>
        </property>
        <property name="toolTip">
         <string>If removed, Mixxx will no longer watch this directory and its subdirectories for new tracks.</string>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="3">
       <widget class="QListView" name="dirList">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="uniformItemSizes">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="PushButtonAddDir">
        <property name="font">
         <font/>
        </property>
        <property name="toolTip">
         <string>Add a directory where your music is stored. Mixxx will watch this directory and its subdirectories for new tracks.</string>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="PushButtonRelocateDir">
        <property name="toolTip">
         <string>If an existing music directory is moved, Mixxx doesn't know where to find the audio files in it. Choose Relink to select the music directory in its new location. &lt;br/&gt; This will re-establish the links to the audio files in the Mixxx library.</string>
        </property>
        <property name="text">
         <string extracomment="This will re-establish the links to the audio files in the Mixxx database if you move an music directory to a new location.">Relink</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Audio File Formats</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Additional Format Plugins:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>MP3, Ogg Vorbis, FLAC, WAVe, AIFF</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Built-in</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButtonExtraPlugins">
        <property name="text">
         <string>Available Online...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Loaded Plugins:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="pluginsLabel">
        <property name="text">
         <string>None</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Miscellaneous</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkbox_ID3_sync">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Synchronize ID3 tags on track modifications</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_library_scan">
        <property name="text">
         <string>Rescan library on start-up</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBox_use_relative_path">
        <property name="text">
         <string>Use relative paths for playlist export if possible</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>External Libraries</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="8" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>You will need to restart Mixxx for these settings to take effect.</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_show_rhythmbox">
        <property name="text">
         <string>Show Rhythmbox Library</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox_show_banshee">
        <property name="text">
         <string>Show Banshee Library</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBox_show_itunes">
        <property name="text">
         <string>Show iTunes Library</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="checkBox_show_traktor">
        <property name="text">
         <string>Show Traktor Library</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>All external libraries shown are write protected.</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>dirList</tabstop>
  <tabstop>PushButtonAddDir</tabstop>
  <tabstop>PushButtonRelocateDir</tabstop>
  <tabstop>PushButtonRemoveDir</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButtonExtraPlugins</tabstop>
  <tabstop>checkBox_library_scan</tabstop>
  <tabstop>checkbox_ID3_sync</tabstop>
  <tabstop>checkBox_use_relative_path</tabstop>
  <tabstop>checkBox_show_rhythmbox</tabstop>
  <tabstop>checkBox_show_banshee</tabstop>
  <tabstop>checkBox_show_itunes</tabstop>
  <tabstop>checkBox_show_traktor</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
